games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -1,4 +1,4 @@
-From ba88fe1ad26fc936fcfece46929b4b1eebb091c1 Mon Sep 17 00:00:00 2001
+From 63e8dd7ba1d48d2f7f235a5015654d26fd36fd3c Mon Sep 17 00:00:00 2001
```

Comment:
If you want to have the smallest git diff, you can replace "63e8dd7ba1d48d2f7f235a5015654d26fd36fd3c" by "ba88fe1ad26fc936fcfece46929b4b1eebb091c1".